src/cs/usfca/edu/histfavcheckout/externalapis/APIClient.java

```diff
@@ -36,7 +36,7 @@ public static SearchMoviesResponse getAllMovies(Set<Integer> movies) {
 		HttpURLConnection con = request.connect(url, "GET");
 		String response = request.getResponse(con);
 		if (response != null) {
-			//TODO: Log this: System.out.println("Received : " + response.toString());
+			System.out.println("Received : " + response.toString());
```

Comment:
As specified earlier. Let's only catch error workflows and not the success ones. Because usually success ones are more frequent than failure ones and we might end up adding more processing time for success workflows.